app/src/main/java/com/trunghoang/aroundhere/data/model/PlacesRepository.java

```diff
@@ -0,0 +1,30 @@
+package com.trunghoang.aroundhere.data.model;
+
+import android.support.annotation.NonNull;
+
+import com.trunghoang.aroundhere.data.remote.PlacesRemoteDataSource;
+
+public class PlacesRepository implements PlacesDataSource {
+    private static PlacesRepository INSTANCE;
+    private PlacesRemoteDataSource mPlacesRemoteDataSource;
+
+    private PlacesRepository(PlacesRemoteDataSource placesRemoteDataSource) {
```

Comment:
```suggestion
    private PlacesRepository(PlacesDataSource remote) {
```
em nên khai báo interface thay vì implement như vậy mới có giá trị